vnext/PropertySheets/React.Cpp.props

```diff
@@ -70,15 +70,19 @@
         WIN32_LEAN_AND_MEAN           - Reduces the Windows API included surface.
       -->
       <PreprocessorDefinitions>
-        BOOST_ALL_NO_LIB=1;
-        WIN32_LEAN_AND_MEAN;
         _SILENCE_ALL_CXX17_DEPRECATION_WARNINGS;
-        BOOST_SYSTEM_SOURCE;
+        BOOST_ALL_NO_LIB=1;
         BOOST_ERROR_CODE_HEADER_ONLY;
+        BOOST_NO_RTTI; 
+        BOOST_NO_TYPEID;
+        BOOST_SYSTEM_SOURCE;
+        GTEST_HAS_RTTI=0;
```

Comment:
I'd also be fine with putting this just in test projects. See pros and cons either way

Reply:
I think it is fine. Since you are changing this code, do you mind to add the WINRT_LEAN_AND_MEAN here too?

---
In reply to: [370667565](https://github.com/microsoft/react-native-windows/pull/3948#discussion_r370667565) [](ancestors = 370667565)

Reply:
Can do